sdv/data_processing/data_processor.py

```diff
@@ -103,6 +104,112 @@ def __init__(self, metadata, enforce_rounding=True, enforce_min_max_values=True,
         if self._primary_key:
             self._keys.append(self._primary_key)
 
+    def _get_column_in_multi_column_transformer(self):
+        """Get the columns that are part of a multi column transformer.
+
+        Returns:
+            list:
+                A list of columns that are part of a multi column transformer.
+        """
+        return [
+            col for col_tuple in self.columns_to_multi_col_transformer for col in col_tuple
+        ]
+
+    def _check_import_address_transformers(self):
+        """Check that the address transformers can be imported."""
+        has_randomlocationgenerator = hasattr(rdt.transformers, 'RandomLocationGenerator')
+        has_regionalanonymizer = hasattr(rdt.transformers, 'RegionalAnonymizer')
+        if not has_randomlocationgenerator or not has_regionalanonymizer:
+            raise ImportError(
+                'You must have SDV Enterprise with the address add-on to use the address features'
+            )
+
+    def _get_columns_in_address_transformer(self):
```

Comment:
this doesn't seem to be used anywhere

Reply:
It is being used in #1614 I had the same confusion.

Reply:
Maybe we should just move it to that PR. It's kind of confusing here